lib/enums/query/props/server_properties.py

```diff
@@ -54,6 +55,9 @@ def get_prop_mapping(prop):
             ServerProperties.FLAVOR_ID: lambda a: a["flavor_id"],
             ServerProperties.IMAGE_ID: lambda a: a["image_id"],
             ServerProperties.PROJECT_ID: lambda a: a["location"]["project"]["id"],
+            ServerProperties.ADDRESSES: lambda a: ", ".join(
+                [ip["addr"] for network in a["addresses"].values() for ip in network]
+            ),
```

Comment:
This should be in a separate method, else we can't easily unit test it and it's non-trivial